src/components/IncidentInfo/IncidentInfoSlice.js

```diff
@@ -16,6 +17,16 @@ export const getIncidentInfo = createAsyncThunk(
   }
 )
 
+export const updateIncidentInfo = createAsyncThunk(
+  'incidentInfo/updateIncidentInfo',
+  async ({id, incidentObj}) => {
+    console.log(id)
```

Comment:
If we have time we should review the Redux Toolkit docs to find out exactly what these different arguments in async represent - good to remember for the future though, you can only pass one argument into async. If you want more than one, you need to destructure